.github/workflows/reusable-module-prep-release.yml

```diff
@@ -70,6 +82,14 @@ jobs:
           path: ${{ steps.composer-cache.outputs.dir }}
           key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
 
+      - name: Cache NPM cache directory
+        uses: actions/cache@5a3ec84eff668545956fd18022155c47e93e2684  # v4.2.3
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-npm-${{ hashFiles('**/package-lock.json') }}
```

Comment:
The actions/setup-node action has caching built into it, and it actually looks like we already have it configured. Is there a reason why we this is being added using the actions/cache action directly here?

Reply:
Thanks for pointing that out. I've removed the npm cache step. I hadn't realized the setup-node did it automatically.